test/test_ssl.rb

```diff
@@ -1222,6 +1222,56 @@ def test_get_ephemeral_key
     end
   end
 
+  def test_fallback_scsv
+    ctx_proc = proc { |ctx|
+        ctx.ssl_version = :TLSv1_2
```

Comment:
Please use #max_version= and #min_version= in new tests. In this case, specifying just max_version would be enough. (This applies to the other instances too.)

Reply:
Fixed